nextflow.config

```diff
@@ -259,6 +283,12 @@ manifest {
 // Load modules.config for DSL2 module specific options
 includeConfig 'conf/modules.config'
 
+// Include extra DSL2 module specific options for tests after loading conf/modules.config to avoid conflicts
+profiles {
+    tools_somatic       { includeConfig 'conf/test_tools_somatic_modules.config' }
+    tools_somatic_ascat { includeConfig 'conf/test_tools_somatic_modules.config' }
```

Comment:
Still don't understand why these have to be put here I'm afraid 😬 Can you give me an example by pointing to some code as to what is happening?

Reply:
> These two profiles point to the same file?

Yes, just not to repeat the same code twice

> Still don't understand why these have to be put here I'm afraid grimacing Can you give me an example by pointing to some code as to what is happening?

So, I did some extensive tests, and basically, it's all due to some priority conflicting over the content of the process scope. Would need some more in-depth tests here, but I suppose some issue when resolving scope between config files/profile etc. and I think it appeared on the more recent Nextflow versions.

I think we actually encountered the issue, and that's why we bumped Nextflow version when we updated TEMPLATE cf (https://github.com/nf-core/sarek/pull/720/commits/7293436f0c511d0236ec1664514128db95461bc7).
Cf bumping back Nextflow version (ie #750 based on this PR works, and #751 based on current dev fails).

But basically, all DSL2 related stuff in `conf/test_tools_somatic_modules.config` has to be loaded after we load the `conf/modules.config`, which is why I moved that out of the `conf/test.config`.

Which in fact does actually makes sense in the end.

Reply:
It could be happening because you need to use the process simple name in the `modules.config` and the fully qualified name in the `test_*` configs. The latter has precedence so might be why you are having these issues.

Reply:
Make sense to use fully qualified name to gain priority over delaying the load of the config

Reply:
Did it work?